apps/condo/domains/notification/schema/RemoteClient.js

```diff
@@ -80,6 +83,16 @@ const RemoteClient = new GQLListSchema('RemoteClient', {
             kmigratorOptions: { null: true },
         },
 
+        pushType: {
+            schemaDoc: 'Represents the type of push to be sent to this remote client like default/silent-data/etc.',
```

Comment:
It's about the internal client display to user push protocol. The remote client can choose the type of work with push messages. 

You need to clarify it here! It's not clear at the moment